swap-orchestrator/src/compose.rs

```diff
@@ -56,6 +59,7 @@ impl From<OrchestratorNetworks<monero::Network, bitcoin::Network>> for Orchestra
                 bitcoind_rpc: 8332,
                 bitcoind_p2p: 8333,
                 electrs: 50001,
+                arti_socks: 9150,
```

Comment:
this should be a constant and not configurable because the arti container does not allow this to be configurable. I think you can move it into a `const` within `swap-orchestrator/src/images.rs`.